### PR TITLE
More choices for overwear clothing

### PR DESCRIPTION
**Pull request by alsoandanswer**
dreammaker_5L7A4YRaPE
dreammaker_gXevdRJw85

changes: 
  - rscadd: "Adds blazers, argyle sweaters, sweater vests, argyle sweater vests, sweater cardigans and argyle sweater cardigans."
  - rscadd: "These new clothing types can be found in the loadout, under new selections. Check the loadout!"
  - tweak: "Makes cardigans and sweaters look better. Check them out."
  - tweak: "Cardigans can now be buttoned and unbuttoned."
  - rscdel: "Killed off the green suit jacket for being practically useless."

also deleted the turtleneck since it was just useless legacy thing

update : added tubenecks and turtlenecks as well as argyle variants

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/clothing/suits/hoodies.dm [hunk 1]
```diff
@@ -205,11 +205,11 @@
 	if(icon_state == icon_open)
 		icon_state = icon_closed
 		item_state = icon_closed
-		to_chat(usr, "You zip the hoodie.")
+		to_chat(usr, "You zip the [src].")
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/hoodies.dm:208]: \the

File: code/modules/clothing/suits/hoodies.dm [hunk 2]
```diff
@@ -205,11 +205,11 @@
...
 	else if(icon_state == icon_closed)
 		icon_state = icon_open
 		item_state = icon_open
-		to_chat(usr, "You unzip the hoodie.")
+		to_chat(usr, "You unzip the [src].")
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/hoodies.dm:212]: \the

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -618,11 +609,35 @@
...
 	item_state = "cardigan"
+	icon_open = "cardigan_open"
+	icon_closed = "cardigan"
+
+/obj/item/clothing/suit/storage/toggle/cardigan/sweater
+	name = "sweater cardigan"
+	desc = "A cozy, warm knit sweater cardigan."
+	desc_fluff = "Half as warm as a sweater, and half as fashionable as a cardigan. Not like it matters for coffee-house dwelling beatniks like yourself."
+	icon_state = "cardigan"
+	item_state = "cardigan"
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/miscellaneous.dm:628]: this and the cardigan above are meant to have the same state?

File: code/modules/clothing/suits/storage.dm [hunk 4]
```diff
@@ -49,11 +49,11 @@
 		if(icon_state == icon_open) //Will check whether icon state is currently set to the "open" or "closed" state and switch it around with a message to the user
 			icon_state = icon_closed
 			item_state = icon_closed
-			to_chat(usr, "You button up the coat.")
+			to_chat(usr, "You button up the [src].")
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/storage.dm:52]: \the

File: code/modules/clothing/suits/storage.dm [hunk 5]
```diff
@@ -49,11 +49,11 @@
...
 		else if(icon_state == icon_closed)
 			icon_state = icon_open
 			item_state = icon_open
-			to_chat(usr, "You unbutton the coat.")
+			to_chat(usr, "You unbutton the [src].")
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/storage.dm:56]: \the

File: code/modules/customitems/item_defines.dm [hunk 6]
```diff
@@ -2128,13 +2128,13 @@ All custom items with worn sprites must follow the contained sprite system: http
...
-	var/obj/item/clothing/under/sweater/G = new(get_turf(user))
-	G.color = ball.color
+	var/obj/item/clothing/accessory/sweater/S = new(get_turf(user))
+	S.color = ball.color
 	qdel(ball)
 	ball = null
 	working = FALSE
 	update_icon()
-	to_chat(user, "<span class='warning'>You finished \the [G]!</span>")
+	to_chat(user, "<span class='warning'>You finished \the [S]!</span>")
```
Comment (NonQueueingMatt) [inline code/modules/customitems/item_defines.dm:2137]: change this to "You finish \the S!"

**Pull request merged**